sls/Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM alpine:3.9
+FROM alpine:3.9.2
```

Comment:
There is no need to pin the patch version, `alpine:3.9` should be the same as `alpine:3.9.2`